echoregions/lines/lines.py

```diff
@@ -55,7 +55,10 @@ def replace_nan_depth(self, inplace: bool = False) -> DataFrame:
         regions["depth"] = regions["depth"].apply(
             lambda x: self._nan_depth_value if x == ECHOVIEW_NAN_DEPTH_VALUE else x
         )
-        return regions
+        if inplace:
+            return
+        else:
+            return regions
```

Comment:
We can just do like below?
```suggestion
        if not inplace:
            return regions
```